english/you-are-in-minecraft/dig_down/goright/goright.md

```diff
@@ -0,0 +1 @@
+You enter a dimly lit chamber and see a tall dark figure. The enderman immediately teleports over to you, smiling, and...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.